src/studio/nuxeo-studio/working-in-studio/vocabularies.md

```diff
@@ -176,13 +176,38 @@ This types has more parameters than the others:
 - **Move up** and **Move down**: Used to move the values (that are ordered).
 - **Set tree depth:** Allows to set more than two degrees of values on a hierarchical vocabulary. Select2 widgets are directly compatible with any levels. If you want to have more than two levels with chain selects, you can read the [Create a n-level select widget how-to]({{page space='nxdoc' page='how-to-create-a-n-level-select-widget'}}).
 
+## Available vocabularies by default
+
+All vocabularies editable from the user interface are listed in the vocabularies management interface.
+
+
+To consult a vocabulary or its entries click on the **Admin > Vocabularies** tab and select the vocabulary you want to consult in the drop down list.
+
+| Vocabulary  | Elements  |
+|---|---|
+| Continent | Lists continents. |
+| Country | Lists countries and parent continents. |
+| Inverse_predicates | Lists inverse predicates that defines relations between documents. JSF only.   |
+| L10ncoverage | Lists countries and their parent continent in english and in french. |
+| L10nsubjects | Lists subjects and their parent subject. |
+| Language | Lists languages. |
+| Nature | Lists several types of documents. |
+| nxtrIndustry  | Lists industries. |
+| Predicates | Lists predicates.  |
```

Comment:
Re - mention the fact that it is for realtions (as inverse predicates) and maybe put it before.

Reply:
It's specified "JSF only", on inverse predicates, but not on predicates. Should we mention on both?